### DISCUSSION TITLE
Deforum for A1111

### DISCUSSION THREAD

**Discussion by Anipsy** [vladmandic/sdnext#206] [General]
Any chance of maintaining compatibility with that A1111 version of Deforum for the fork?
When running it right out the box after fresh installation in the fork it always returns the same error after rendering 2nd frame no matter what settings i use/change:
[error](https://i.imgur.com/H3Xf9df.png)

**Comment by vladmandic** [comment]
i'll take a closer look, but upon first look this seems like deforum may not be compatible with some of the newer libraries specified in requirements used by this repo.
